libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -231,6 +250,7 @@ SRV_Channels::copy_radio_in_out(SRV_Channel::Aux_servo_function_t function, bool
             if (do_input_output) {
                 rc->read();
             }
+            set_output_unlimited_once(function);
```

Comment:
I understand why you did this, but I wouldn't. If the radio has a bigger range, in my opinion, it should be limited to the configured output range.